install_files/ansible-base/securedrop-configure.yml

```diff
@@ -60,6 +60,7 @@
 
     - name: securedrop_app_gpg_fingerprint
       prompt: Full fingerprint for the SecureDrop Application GPG Key
+      default: ' '
```

Comment:
if you put `''` it doesn't register with ansible and doesnt get inserted into the `site-specific` file.